components/gcp/container/component_sdk/python/kfp_component/google/bigquery/_query.py

```diff
@@ -24,6 +24,40 @@
 # TODO(hongyes): make this path configurable as a environment variable
 KFP_OUTPUT_PATH = '/tmp/kfp/output/'
 
+
+def query_only(query, project_id, output_path, dataset_location='US', job_config=None):
+    """Submit a query to Bigquery service and dump outputs to Bigquery table or 
```

Comment:
```suggestion
    """Submits a query to Bigquery service and dump outputs to Bigquery table or 
```

See http://google.github.io/styleguide/pyguide.html#383-functions-and-methods, descriptive-style docstring is preferred.

Reply:
Thanks for the info, I removed this function after feedback.